gulpfile.js

```diff
@@ -283,3 +285,5 @@ gulp.task(
     'deploy-demo', function(done) { runSequence('clean:demo', 'build:demo', 'demo-push', 'clean:demo-cache', done); });
 
 gulp.task('default', function(done) { runSequence('lint', 'enforce-format', 'ddescribe-iit', 'test', done); });
+
+gulp.task('ci', function(done) { runSequence('default', 'build:demo', done); });
```

Comment:
Oh man, it should be part of this PR. Good catch, thnx. Will add and merge if Travis gets green.